website/django/data/searchtest.py

```diff
@@ -254,12 +216,12 @@ def test_request_without_release_number(self):
 
     #Filter testing: Gene type, pathogenicity, and source selection are all filter options
     def test_filter_by_gene_type_brca_1(self):
-        '''Tests filtering by 'Gene' option by BRCA1'''
+        '''Tests gene filter to ensure only variants with the gene specified in the request are returned'''
         #creates a new BRCA2 variant
-        new_variant_brca2 = test_data.new_variant()
-        new_variant_brca2["Gene_Symbol"] = "BRCA2"
-        new_variant_brca2["Genomic_Coordinate_hg38"] = "chr13:111111:A>G"
-        (new_variant_brca2, new_current_variant_brca2) = create_variant_and_materialized_view(new_variant_brca2)
+        new_variant_brca1 = test_data.new_variant()
```

Comment:
this should include the creation of 2 variants, one with gene_symbol brca1 and one with gene_symbol brca2, then it should ensure only the variants with the requested gene_symbol is returned. if there are no variants with gene_symbol brca2 in the database, the test doesn't tell us anything useful. also, shouldn't it get back two variants for brca1 since existing_variant also has a gene_symbol of brca1?